Sumemos/sumemos.js

```diff
@@ -0,0 +1,33 @@
+function isanumber (orden_num) {
+    do {
+        input_user = parseFloat(prompt("Ingresa el " + orden_num + " número"));
+    }
+    while (isNaN(input_user));   
+
+    return input_user;
+}
+
+let numbers = [];
+alert("Programa para sumar cinco números");
+
+/* numbers[0] = isanumber("primer");
+numbers[1] = isanumber("segundo");
+numbers[2] = isanumber("tercer");
+numbers[3] = isanumber("cuarto");
+numbers[4] = isanumber("quinto"); */
+
+const msj_user = ["primer", "segundo", "tercer", "cuarto", "quinto"]; 
+
+for (i = 0; i < 5; i++) {
```

Comment:
La variable _**i**_ se debe indicar como **_let i_**, recuerda que los nombres de las variables deben ser más descriptivos. Los valos constantes en este caso el **_5_** es considerado un **magic number**, trata de evitar este tipo de valores, para este caso es mejor guardar el tamaño del arreglo en una variable. El rendimiento de la instrucción **_i++_** no es tan bueno, mejor utiliza la instrucción **_i += 1_**. El for te quedaría de la siguiente manera: 

> `for (let index = 0, msjUserLength = msj_user.length; index < msjUserLength; index += 1)`

Reply:
Ok! cambiado.